index.bs

```diff
@@ -8960,24 +8963,179 @@ The [=remote end steps=] given |session|, and |command parameters| are:
 
 </div>
 
+#### The input.interceptFileDialogs Command ####  {#command-input-interceptFileDialogs}
+
+The <dfn export for=commands>input.interceptFileDialogs</dfn> command
+enables/disables interception of file dialogs emitted within a given browsing
+context.
+
+<dl>
+   <dt>Command Type</dt>
+   <dd>
+    <pre class="cddl remote-cddl">
+      input.InterceptFileDialogs = (
+        method: "input.interceptFileDialogs",
+        params: input.InterceptFileDialogsParameters
+      )
+
+      input.InterceptFileDialogsParameters = {
+         ? enabled: bool .default true,
+         ? contexts: [*browsingContext.BrowsingContext],
+      }
+    </pre>
+   </dd>
+   <dt>Return Type</dt>
+   <dd>
+    <pre class="cddl">
+      EmptyResult
+    </pre>
+   </dd>
+</dl>
+
+Each [=top-level browsing context=] has a <dfn>file dialog interception
+flag</dfn>, which is false unless otherwise stated.
+
+Each [=top-level browsing context=] has a <dfn>current file dialog</dfn>, which is
+the [=Attr/element=] corresponding to the current [=show the picker|file picker=]
+prompt, if any. Unless stated otherwise, this is [=null=].
+
+<div algorithm="remote end steps for input.interceptFileDialogs">
+
+The [=remote end steps=] given |command parameters| are:
+
+1. Let the |list of contexts| be the value of the <code>contexts</code> field of
+   |command parameters| if it is present or null if it isn't.
+
+1. Let |enabled| be the value of the <code>enabled</code> field of |command
+   parameters|.
+
+1. If |enabled| is null, set |enabled| to true.
+
+1. For each |context| in |list of contexts|:
+
+   1. If |context| is not a [=top-level browsing context=], return [=error=] with
```

Comment:
In this method too, I think it does not need to be restricted to top-level contexts